api/settings.md

```diff
@@ -107,6 +107,17 @@ Status: 200 OK
     "force_update": {
       "enabled": false
     }
+  },
+  "provider": {
+    "providerkey": {
+      "image": "http://provider/image.png",
+      "thumbnail": "http://provider/thumbnail.png",
+      "header": "[?]Please provide the data necessary for booking the parking spaces.",
+      "legal": {
+        "accept": ["http://www.provider.com/terms_and_conditions.html"],
+        "shown": ["http://www.provider.com/terms_and_conditions_info.html"]
+      }
+    }
   }
```

Comment:
Oh man for this example it would have been so nice to have parku as a provider too... I can see us changing this stuff in the future, when someone uses our API and wants to retrieve our terms and conditions for handing them to a user...